appium/tests/data/index.ts

```diff
@@ -212,5 +212,8 @@ export const CommonData = {
   revokedMockUser: {
     email: 'revoked@domain.test',
     name: 'revoekd@flowcrypt.com'
+  },
+  keyManagerURL : {
+    link: 'http://127.0.0.1:8001/ekm'
```

Comment:
`link` doesn't seem to correct naming. Should be `mockLink` or `mockServer`.
There is also live ekm url (`https://ekm.flowcrypt.com`)